src/service/poc_mobile.proto

```diff
@@ -4,6 +4,21 @@ package helium.poc_mobile;
 
 import "mapper.proto";
 
+message subscriber_event_req_v1 {
```

Comment:
The message name here is generic, what is we need to incorporate additional subscriber requests such as subscriber mapping requests
Suggest renaming `subscriber_location_req_v1`